addons/point_of_sale/models/pos_session.py

```diff
@@ -737,8 +737,11 @@ def action_show_payments_list(self):
             'name': _('Payments'),
             'type': 'ir.actions.act_window',
             'res_model': 'pos.payment',
-            'view_id': self.env.ref('point_of_sale.view_pos_payment_tree').id,
-            'view_mode': 'tree',
+            'views': [
```

Comment:
you can remove 'views', 'view_mode' is enough as it will take the views with lowest priority and there is only one tree view and one form view